Mathlib/GroupTheory/SpecificGroups/Alternating/MaximalSubgroups.lean

```diff
@@ -0,0 +1,538 @@
+/-
+Copyright (c) 2025 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+module
+
+public import Mathlib.GroupTheory.GroupAction.SubMulAction
+public import Mathlib.GroupTheory.Perm.MaximalSubgroups
+public import Mathlib.GroupTheory.SpecificGroups.Alternating
+
+/-! # Maximal subgroups of the alternating group
+
+* `AlternatingGroup.isCoatom_stabilizer`: if neither `s : Set α` nor its complement is empty,
+and if, moreover, `Nat.card α ≠ 2 * s.ncard',
+then `stabilizer (alternatingGroup α) s` is a maximal subgroup of `alternatingGroup α`.
+
+This is the “intransitive case” of the O'Nan-Scott classification
+of maximal subgroups of the alternating groups.
+
+Compare with `Equiv.Perm.isCoatom_stabilizer` for the case of the permutation group.
+
+## TODO
+
+  * Application to primitivity of the action
+    of `alternatingGroup α` on finite combinations of `α`.
+
+  * Formalize the other cases of the classification.
+    The next one should be the *imprimitive case*.
+
+## Reference
+
+The argument is taken from [M. Liebeck, C. Praeger, J. Saxl,
+*A classification of the maximal subgroups of the finite
+alternating and symmetric groups*, 1987][LiebeckPraegerSaxl-1987].
```

Comment:
I am not sure about the md formatting of multi-line references: it may work, but it is something to keep in mind!